ntoskrnl/io/pnpmgr/pnpmgr.c

```diff
@@ -1781,11 +1861,16 @@ IopQueryHardwareIds(PDEVICE_NODE DeviceNode,
                               &Stack);
    if (NT_SUCCESS(Status))
    {
-      /*
-       * FIXME: Check for valid characters, if there is invalid characters
-       * then bugcheck.
-       */
+      IsValideID = IopValidateID((PWCHAR)IoStatusBlock.Information,
+                                 MAX_DEVICE_ID_LEN,
+                                 MAX_SEPARATORS_MULTISZ,
+                                 TRUE);
+
+      if (!IsValideID)
```

Comment:
Please use braces around all if/else blocks

Reply:
Is this a general rule for code style or is it only in this case?

Reply:
It's a general rule, at least inside ntoskrnl.

Reply:
Sound Ok. Changed